### ISSUE TITLE
Why is the accuracy of svt so low?Is it over fitted?

### ISSUE THREAD

**Issue by dzyanshan** [liuch37/sar-pytorch#9]

**Comment by liuch37** [comment]
Note text recognition requires very large amount of data to have good accuracy. In my experiments I only use a single original  dataset for training, not mixing datasets together. SVT has only 350 images here, therefore you can imagine highly overfitting can happen. But you can see from there if increasing number of training data, the accuracy increases. These experiments are just for sanity check.

**Comment by dzyanshan** [comment]
thank you!
